### PR TITLE
feat: add a flag for setting the version template vs scanning files

### PR DESCRIPTION
**Pull request by polothy**
Example usage:
```
jx-release-version -version=0.2.0-SNAPSHOT
```

### CODE REVIEW

**Comment by ccojocar** [comment]
@polothy Please can you rebase this pull request? Thanks!

**Comment by polothy** [comment]
@ccojocar rebase done, thanks

File: main.go [hunk 1]
```diff
@@ -52,6 +53,7 @@ func main() {
 	dir := flag.String("folder", ".", "the folder to look for files that contain a pom.xml or Makfile with the project version to bump")
 	owner := flag.String("gh-owner", "", "a github repository owner if not running from within a git project  e.g. fabric8io")
 	repo := flag.String("gh-repository", "", "a git repository if not running from within a git project  e.g. fabric8")
+	ver := flag.String("version", "", "use this instead of Makefile, pom.xml, etc, e.g. -version=2.0.0-SNAPSHOT")
 	samerelease := flag.Bool("same-release", false, "for support old releases: for example 7.0.x and tag for new realese 7.1.x already exist, with `-same-release` argument next version from 7.0.x will be returned ")
 	version := flag.Bool("version", false, "prints the version")
```
Comment (ccojocar) [inline main.go]: I think you version flag collides now with the boolean `version` flag which is intended to just print the current version of the tool. Probably it needs another name.

Comment (polothy) [reply main.go]: Oh, didn't see that!  Any suggestions? Otherwise, I'll try to come up with something in a bit...

File: main.go [hunk 2]
```diff
@@ -44,20 +44,22 @@ type config struct {
...
-	dir := flag.String("folder", ".", "the folder to look for files that contain a pom.xml or Makfile with the project version to bump")
+	dir := flag.String("folder", ".", "the folder to look for files that contain a pom.xml or Makefile with the project version to bump")
 	owner := flag.String("gh-owner", "", "a github repository owner if not running from within a git project  e.g. fabric8io")
 	repo := flag.String("gh-repository", "", "a git repository if not running from within a git project  e.g. fabric8")
-	samerelease := flag.Bool("same-release", false, "for support old releases: for example 7.0.x and tag for new realese 7.1.x already exist, with `-same-release` argument next version from 7.0.x will be returned ")
-	version := flag.Bool("version", false, "prints the version")
+	baseVersion := flag.String("base-version", "", "use this instead of Makefile, pom.xml, etc, e.g. -base-version=2.0.0-SNAPSHOT")
+	samerelease := flag.Bool("same-release", false, "for support old releases: for example 7.0.x and tag for new release 7.1.x already exist, with `-same-release` argument next version from 7.0.x will be returned ")
+	ver := flag.Bool("version", false, "prints the version")
```
Comment (polothy) [inline main.go]: FYI, renamed this var because it conflicted with imported package name.

**Comment by polothy** [comment]
Ready for re-review

**Comment by polothy** [comment]
Thanks dude!

**Pull request merged**